components/Navigation.tsx

```diff
@@ -15,24 +22,66 @@ export type StackNavigatorType<
   ParamsList extends Record<string, object | undefined>
 > = ReturnType<typeof createStackNavigator<ParamsList>>
 
+/**
+ * Returns an object of functions to navigate to essential app screens.
+ */
+export const useCoreNavigation = () => {
+  // TODO: - Actually Navigate
+  const navigation = useNavigation()
+  return {
+    presentEditEvent: (edit: EditEventFormValues, id?: EventID) => {
+      const formValues = toRouteableEditFormValues(edit)
```

Comment:
So this takes everything needed for an event (title, description, etc.) and adds a startDateTime?

Reply:
Yes, it's just that we're not allowed to pass `Date` as a navigation parameter for react-navigation, so we need a function to serialize `startDateTime` to a string such that we don't get the warning message.